cli/auth.go

```diff
@@ -128,7 +128,8 @@ var AuthApiInfoToken = &cli.Command{
 
 		// TODO: Log in audit log when it is implemented
 
-		fmt.Printf("%s=%s:%s\n", cliutil.EnvForRepo(t), string(token), ainfo.Addr)
+		currentEnv, _ := cliutil.EnvsForRepo(t)
+		fmt.Printf("%s=%s:%s\n", currentEnv, string(token), ainfo.Addr)
```

Comment:
We have a bug here, the resulting string looks like:
```
[MINER_API_INFO]=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkIiwid3JpdGUiLCJzaWduIiwiYWRtaW4iXX0.X0ujyPv7C1O-Xy88jpFnaPQDUeKdcYHtVvPqlFuzB_0:/ip4/127.0.0.1/tcp/2345/http
```

and it should look like
```
MINER_API_INFO=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkIiwid3JpdGUiLCJzaWduIiwiYWRtaW4iXX0.X0ujyPv7C1O-Xy88jpFnaPQDUeKdcYHtVvPqlFuzB_0:/ip4/127.0.0.1/tcp/2345/http
```

Reply:
So probably just picking the first item from that slice is good.

Reply:
Fixed by changing the signature of EnvForAPIInfos to return a single primary var.